### PR TITLE
Add DLDAQ example

### PR DESCRIPTION
**Pull request by chancesmith**
documentNumber wasn't available if the data had `DLDAQ` in the ANSI line.

### CODE REVIEW

File: spec/parseUsdl.spec.js [hunk 1]
```diff
@@ -18,6 +19,11 @@ describe('USDL Parser', () => {
     expect(parsedData.sex).toBe('F')
   })
 
+  it('should correctly identify DL', () => {
+    const parsedData = parse(UsdlData3, {suppressErrors: true})
+    expect(parsedData.documentNumber).toBe('099964088')
+  })
```
Comment (chancesmith) [inline spec/parseUsdl.spec.js:25]: add sample data and test to match

File: spec/sample/data_3.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+exports.UsdlData3 = `@
+
+ANSI 636053060002DL00410258ZT02990037DLDAQ099964088
```
Comment (chancesmith) [inline spec/sample/data_3.js:3]: DLDAQ provided

File: src/parseUsdl.js [hunk 3]
```diff
@@ -13,13 +13,21 @@ exports.parse = function parseCode128(str, options = defaultOptions) {
     if (!started) {
       if (line.indexOf('ANSI ') === 0) {
         started = true
+
+        // has DLDAQ
+        if(line.indexOf('DLDAQ') !== -1) {
+          const lineArray = line.split('DLDAQ')
+          line = 'DAQ' + lineArray[1]
+        }
```
Comment (chancesmith) [inline src/parseUsdl.js:21]: check if `DLDAQ` is in the ANSI line and modify the line for the parse utils to add to the props object

File: src/parseUsdl.js [hunk 4]
```diff
@@ -13,13 +13,21 @@ exports.parse = function parseCode128(str, options = defaultOptions) {
     if (!started) {
       if (line.indexOf('ANSI ') === 0) {
         started = true
+
+        // has DLDAQ
+        if(line.indexOf('DLDAQ') !== -1) {
```
Comment (mvayngrib) [inline src/parseUsdl.js:18]: slightly simpler, self-explanatory (non-blocking)
```suggestion
        if(line.includes('DLDAQ')) {
```

**Review by mvayngrib — APPROVED** [review]
thanks @chancesmith! I left a small non-blocking suggestion, lmk if u want to get it in, otherwise we can merge as is

question: how did you run across this? And is it documented anywhere?

**Comment by chancesmith** [comment]
> thanks @chancesmith! I left a small non-blocking suggestion, lmk if u want to get it in, otherwise we can merge as is
> 
> question: how did you run across this? And is it documented anywhere?

I'm no expert here, but this is what I found that matched the ID we were having trouble with.

On page 149, of the ID standard doc, shows the DLDAQ
https://www.aamva.org/FL-ITN-DriverRelatedCredentialPrintSolutionExh3-022016/

Screen Shot 2021-12-07 at 9 36 05 AM

**Comment by mvayngrib** [comment]
@chancesmith closed by accident? Or do you no longer want to get this in?

**Comment by chancesmith** [comment]
Oops - I totally hit the wrong button

File: src/parseUsdl.js (hunk 4 above)
Comment (chancesmith) [reply src/parseUsdl.js:18]: love it - I'll push a change for this

**Comment by chancesmith** [comment]
added `includes` and tests are passing
Screen Shot 2021-12-07 at 10 06 53 AM

**Comment by mvayngrib** [comment]
published as parse-usdl@3.2.0

**Pull request merged**